src/pages/Hours.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import 'antd/dist/antd.css';
+import ReviewHours from '../components/ReviewHours';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../components/ReviewHours"